RoosterPlanner.Api/Startup.cs

```diff
@@ -34,44 +35,48 @@ public Startup(IHostingEnvironment env)
-            }).AddJwtBearer(jwtOptions => {
-                jwtOptions.Authority = $"https://login.microsoftonline.com/tfp/{Configuration["AzureAuthentication: TenantId"]}/{Configuration["AzureAuthentication:SignUpSignInPolicyId"]}/";
-                jwtOptions.TokenValidationParameters = new TokenValidationParameters {
-                    ValidateIssuer = false,
-                    ValidAudiences = new List<string> {
-                        Configuration["TokenValidation:ClientIdWeb"],
-                        Configuration["TokenValidation:ClientIdHook"]
-                    }
-                };
-                jwtOptions.Audience = "c832c923-37c6-4145-8c75-a023ecc7a98f";
-                jwtOptions.Events = new JwtBearerEvents {
-                    OnAuthenticationFailed = AuthenticationFailedAsync
-                };
-            });
-
+            services.AddAuthentication(options => { options.DefaultScheme = JwtBearerDefaults.AuthenticationScheme; })
+                .AddJwtBearer(jwtOptions =>
+                {
+                    jwtOptions.Authority =
+                        $"https://login.microsoftonline.com/tfp/{Configuration["AzureAuthentication: TenantId"]}/{Configuration["AzureAuthentication:SignUpSignInPolicyId"]}/";
+                    jwtOptions.TokenValidationParameters = new TokenValidationParameters
+                    {
+                        ValidateIssuer = false,
+                        ValidAudiences = new List<string>
+                        {
+                            Configuration["TokenValidation:ClientIdWeb"],
+                            Configuration["TokenValidation:ClientIdHook"]
+                        }
+                    };
+                    jwtOptions.Audience = "c832c923-37c6-4145-8c75-a023ecc7a98f";
+                    jwtOptions.Events = new JwtBearerEvents
+                    {
+                        OnAuthenticationFailed = AuthenticationFailedAsync
+                    };
+                });
+            services.AddCors();
             services.AddAuthorization();
 
             // Enable Application Insights telemetry collection.
             services.AddApplicationInsightsTelemetry();
 
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2)
-                .AddJsonOptions(options => {
-                options.SerializerSettings.NullValueHandling = NullValueHandling.Include;
-                options.SerializerSettings.ReferenceLoopHandling = ReferenceLoopHandling.Ignore;
-                options.SerializerSettings.DateParseHandling = DateParseHandling.DateTime;
-                options.SerializerSettings.DateFormatHandling = DateFormatHandling.IsoDateFormat;
-                options.SerializerSettings.DateTimeZoneHandling = DateTimeZoneHandling.Utc;
+
+            services.AddRazorPages().AddJsonOptions(options =>
+            {
+                options.JsonSerializerOptions.IgnoreNullValues = false;
             });
```

Comment:
In de API layer hebben we geen RazorPges nodig. Kijk even of je deze nog kan vervangen naar:
services.AddControllers().AddJsonOptions(options => {
                options.JsonSerializerOptions.IgnoreNullValues = false;
                options.JsonSerializerOptions.PropertyNamingPolicy = JsonNamingPolicy.CamelCase;
            });

Reply:
Done